### DISCUSSION TITLE
Sqanti rescue: error in plotting

### DISCUSSION THREAD

**Discussion by ChrissiKalk97** [ConesaLab/SQANTI3#368] [Q&A]
Hello :)

I am using Sqanti3 to clean up my transcriptome assembly and the QC and filter (rules) step have worked really nicely so far. For the Sqanti rescue step I run into the following error:

Generating common filter plots...

`summarise()` has grouped output by 'structural_category'. You can override
using the `.groups` argument.
`summarise()` has grouped output by 'filter'. You can override using the
`.groups` argument.
`summarise()` has grouped output by 'filter'. You can override using the
`.groups` argument.
Warning message:
There was 1 warning in `dplyr::mutate()`.
ℹ In argument: `filter = factor(filter) %>% forcats::fct_relevel(c("Before",
  "After"))`.
Caused by warning:
! 2 unknown levels in `f`: Before and After 
`summarise()` has grouped output by 'filter'. You can override using the
`.groups` argument.
Error in `dplyr::mutate()`:
ℹ In argument: `ism_type = `%>%`(...)`.
Caused by error in `forcats::fct_relevel()`:
! `.f` must be a factor or character vector, not an empty logical
  vector.
Backtrace:
     ▆
  1. ├─... %>% ...
  2. ├─dplyr::mutate(...)
  3. ├─dplyr:::mutate.data.frame(...)
  4. │ └─dplyr:::mutate_cols(.data, dplyr_quosures(...), by)
  5. │   ├─base::withCallingHandlers(...)
  6. │   └─dplyr:::mutate_col(dots[[i]], data, mask, new_columns)
  7. │     └─mask$eval_all_mutate(quo)
  8. │       └─dplyr (local) eval()
  9. ├─... %>% forcats::fct_relevel(c("Unique", "Multiple"))
 10. └─forcats::fct_relevel(., c("Unique", "Multiple"))
 11.   └─forcats:::check_factor(.f)
 12.     └─cli::cli_abort(...)
 13.       └─rlang::abort(...)
Execution halted

I have checked many times that my input files are correct. I run in full mode and with the rules option. Also the QC run on the reference ran without errors. Maybe someone has encountered the same error. Any help and suggestions are much appreciated! :)

Best,

Christina

**Comment by dorotheavellame** [comment]
I am having the same issue, did you find a solution to this? 😅
